challenges/objects-arrays.js

```diff
@@ -61,11 +99,59 @@ The resulting contact information strings should have a space between the first
 
 Log the result of your new array. */
 const contactInfo = [];
+graduates.map((item) => contactInfo.push(`${item.first_name} ${item.email}`));
```

Comment:
remember the differences between methods. what does map automatically do at the end? 

Why would the below code made more sense? 
const newArray = graduates.map((item) => `${item.first_name} ${item.email}`);
console.log(newArray);